src/frameserver/PointGreyCam.cpp

```diff
@@ -158,7 +158,7 @@ void PointGreyCam<T>::configure(const po::variables_map &vm)
         vm, config_table, "index", index, 0, num_cams - 1);
 
     connectToCamera(index);
-    //turnCameraOn();
```

Comment:
I don't know why I got rid of this. For this refactor, I did remove a lot of functionality and then add it back bit, so maybe that's why I forgot this. Anyway, It should not be commented out.